app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -98,6 +98,9 @@ data class Patient (
     @Query("DELETE FROM patient")
     abstract fun clear()
 
+    @Query("SELECT COUNT(uuid) FROM Patient WHERE syncStatus = :syncStatus")
+    abstract fun recordsCountForSyncStatus(syncStatus: SyncStatus) : Flowable<Int>
```

Comment:
1. There's already a function called `patientCount()`. Can we keep the naming consistent?
2. Do you want to overload `patientCount(SyncStatus)` with an extra argument? Do we need `ForSyncStatus` in the name?

Reply:
Done.